features/shipping/managing_shipping_categories/editing_shipping_category.feature

```diff
@@ -9,12 +9,12 @@ Feature: Editing shipping method
         And the store has "Standard" shipping category
         And I am logged in as an administrator
 
-    @ui
+    @ui @api
     Scenario: Seeing disabled code field when editing shipping category
         When I modify a shipping category "Standard"
         Then the code field should be disabled
```

Comment:
We could change it, as it's done [here](https://github.com/Sylius/Sylius/pull/11144/files#diff-aaa1fbb29e9e5f65c6e953a32c8699adR25)